homeassistant/components/sensor/__init__.py

```diff
@@ -190,8 +190,9 @@ class SensorDeviceClass(StrEnum):
     # volume (VOLUME_*)
     VOLUME = "volume"
 
-    # weight (g, kg, mg, µg, oz, lb)
+    # weight/mass (g, kg, mg, µg, oz, lb)
     WEIGHT = "weight"
+    """Using weight instead of mass because is fits better with every day language"""
```

Comment:
Incorrect grammar. If we're writing a docstring we can add a body to the docstring.

Suggestion:
```py
"""Represent a measurement of an object's mass.

Weight is used instead of mass to fit with every day language.
"""
```

Reply:
Opened https://github.com/home-assistant/core/pull/79201